engine/apps/alerts/tests/test_channel_filter.py

```diff
@@ -91,3 +91,34 @@ def test_channel_filter_select_filter_jinja2(make_organization, make_alert_recei
         alert_receive_channel, raw_request_data, force_route_id=channel_filter.pk
     )
     assert satisfied_filter == channel_filter
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    "filtering_term,should_match",
+    [
+        ('{{ "foo" in labels.keys() }}', True),
+        ('{{ "bar" in labels.values() }}', True),
+        ('{{ "bar" in labels.values() or payload["value"] == 5 }}', True),
+        ('{{ "baz" in labels.values() }}', False),
+    ],
```

Comment:
Let's check the cases `{{ labels.foo=="bar"}}`, `{{ labels.foo=="bar" and labels.bar=="baz" }}`, `{{ labels.foo=="bar" or labels.bar=="baz" }}`

I think thats what will be generated by the UI template builder

Reply:
good call 👍